web/src/components/nav/drawer.js

```diff
@@ -44,20 +54,46 @@ export default function Drawer({ isOpen, onDrawerClose }) {
         </ListItem>
       </List>
       <Divider />
-      <List>
-        <ListItem button>
-          <ListItemIcon>
-            <InputIcon />
-          </ListItemIcon>
-          <ListItemText primary={'Sign In'} />
-        </ListItem>
-        <ListItem button>
-          <ListItemIcon>
-            <PersonAddIcon />
-          </ListItemIcon>
-          <ListItemText primary={'Sign Up'} />
-        </ListItem>
-      </List>
+      {isLoggedIn ? (
+        <List>
+          <ListItem
+            button
+            onClick={(e) => {
+              e.preventDefault()
+              logout()
+            }}
```

Comment:
We might want to encapsulate this somehow, but we'll want to use `router.push` when we are going to an internal link to prevent the page from reloading. Even though we're using SSR, we're able to use the standard SPA method of pushing pages without a reload. The browser will hit our Next.js server, render the page and then send down the contents to our browser without a reload. It's super nice.